src/App.vue

```diff
@@ -109,6 +112,15 @@ export default {
       // Then check if btc is operational
       if (this.loadingProgress <= 60) {
         this.loadingText = "Loading Bitcoin Core...";
+
+        // Warn users against pulling power if Core is taking a while
+        const bitcoinSlowDelay = 10 * SECONDS_IN_MS;
+        if (!this.bitcoinStarted) {
+          this.bitcoinStarted = Date.now();
```

Comment:
Is it ok to switch between variable types? 'Cause in the initial state `bitcoinStarted` is set to `false`, but then it's set to an integer here? Shall we not set `bitcoinStarted` to just `0` in the initial state?

Reply:
There is no downside and the behaviour is the same. But yeah, seems neater to keep them the same.